README.md

```diff
@@ -2,12 +2,19 @@
 
 Dejavu is a free, open source implementation of [Game Maker]. It is designed to be compatible with classic versions (8.0 and earlier) and eventually more recent versions (Game Maker: Studio). The goal is to preserve some of indie game dev history, and port it to modern platforms.
 
+You can **[try it out in your web browser]** right now!
+
+The current functionality is mainly focussed on GML support.
+
 [game maker]: https://en.wikipedia.org/wiki/GameMaker_Studio
+[try it out in your web browser]: https://dejavu.abubalay.com/
 
 # Contributing
 
 Dejavu is still in an early stage, but contributions are welcome in the form of bug reports, pull requests, suggestions, and other feedback.
 
+It currently uses Rust nightly, but may move to Rust stable in future.
+
```

Comment:
```suggestion
In its current state, it can compile and run GML and D&D, with some support for binding to APIs implemented in Rust.

Dejavu currently requires a nightly Rust toolchain to build, though mostly out of convenience. It should move to stable Rust in the future.
```

Reply:
> In its current state, it can compile and run GML and D&D, with some support for binding to APIs implemented in Rust.

Should the “The current functionality is mainly focussed on GML support.” above go, then?

Reply:
Yeah, I removed it in the suggestion.

Reply:
Oh, sorry, I'm not used to GitHub's suggested replies system so I didn't notice. Thanks.